src/tests/ftest/launch.py

```diff
@@ -1023,7 +1025,7 @@ def install_debuginfos():
     # installation
     path = os.path.sep + os.path.join('usr', 'share', 'spdk', 'include')
     if os.path.islink(path):
-        cmds = [["sudo", "rm", "-f", path]]
+        cmds.append = [["sudo", "rm", "-f", path]]
```

Comment:
```suggestion
        cmds.append(["sudo", "rm", "-f", path])
```
If you want to append the list to cmds.

Reply:
Yeah, absolutely.  I have no idea what was going through my mind when I made that change.  Clearly incorrect.